blocks/init/src/Blocks/custom/tabs/assets/tabber.js

```diff
@@ -0,0 +1,30 @@
+export class Tabber {
+    constructor(options) {
+        this.tabsElement = options.tabsElement;
+        this.options = options;
+        this.tabsElement.querySelector(`.${this.options.tabJsClass}`).classList.add(this.options.tabVisibleClass);
+        this.tabPanel = this.tabsElement.querySelector(`.${this.options.tabPanelJsClass}`); 
```

Comment:
You can check how the other classes look like in JS. Some practice that we've been doing is to pass the options inside, and then in the body of the constructor assign other properties of the class.